src/main/java/adris/altoclef/commands/GetCommand.java

```diff
@@ -25,7 +26,7 @@ protected void Call(AltoClef mod, ArgParser parser) throws CommandException {
         int count = parser.Get(Integer.class);
 
         if (TaskCatalogue.taskExists(resourceName)) {
-            Task targetTask = TaskCatalogue.getItemTask(resourceName, count);
+            Task targetTask = new GetResourceTask(resourceName, count);
```

Comment:
This change should be undone after you follow the next review comment. (Otherwise would be a valid solution)